app/services/finance/ecf/mentor/output_calculator.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+module Finance
+  module ECF
+    module Mentor
+      class OutputCalculator
+        attr_reader :statement
+
+        def initialize(statement:)
+          @statement = statement
+        end
+
+        def output_breakdown
+          @output_breakdown ||= declaration_types.map do |declaration_type|
+            current_output_for_declaration_type(declaration_type)
+          end
+        end
+
+        def fee_for_declaration(type:)
+          percentage = case type
+                       when :started
+                         started_event_percentage
+                       when :completed
+                         completed_event_percentage
+                       end
+
+          percentage * statement.mentor_contract.payment_per_participant
+        end
+
+      private
+
+        def started_event_percentage
+          0.5
```

Comment:
Would these be better as constants? I've also not seen the `event` terminology before, would something like `STARTED_DECLARATION_FEE_PERCENTAGE = 0.5` make more sense? or maybe wrap it up with declaration types to kill another method, such as:

```
DECLARATION_TYPE_FEE_PROPORTIONS = {
  started: 0.5,
  completed: 0.5.
}
```